packages/docusaurus-plugin-ideal-image/src/index.ts

```diff
@@ -14,16 +14,18 @@ export default function (
   _context: LoadContext,
   options: PluginOptions,
 ): Plugin<void> {
-  const isProd = process.env.NODE_ENV === 'production';
-
   return {
     name: 'docusaurus-plugin-ideal-image',
 
     getThemePath() {
-      return path.resolve(__dirname, './theme');
+      return path.resolve(__dirname, '../lib/theme');
```

Comment:
as @Josh-Cena said we should have a `lib-next` output folder otherwise users swizzling this comp will get a bad output, + we should add a `getTypeScriptThemePath()` to swizzle this comp with `--typescript` option

Reply:
I actually did not assume that this component would be available for swizzling. That's why I rolled back the changes related with the migration to TS. Better to do it in separate PR, right now we don't have proper infrastructure set up for it.